frontend/components/DataSet/_styles.scss

```
@@ -23,6 +23,13 @@
     align-items: baseline;
   }
 
+  &--multiline dd {
+    white-space: normal;
+    overflow: visible;
+    text-overflow: clip;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)